ios/RNWalletModule/RNWalletModule.m

```diff
@@ -28,6 +28,18 @@ @implementation RNWalletModule
 	callback(@[@([PKAddPassesViewController canAddPasses])]);
 }
 
+RCT_EXPORT_METHOD(
+	passes:(RCTResponseSenderBlock)callback
+) {
+    self.passLibrary = [[PKPassLibrary alloc] init];
+    NSArray *passes = [self.passLibrary passes];
+
+    NSDictionary *dict = [NSDictionary dictionaryWithObjects:passes forKeys:[passes valueForKey:@"serialNumber"]];
+    NSArray*serials=[dict allKeys];
+
+    callback(@[[NSNull null], serials]);
```

Comment:
We don't use the Node convention of passing null instead on an error in this project.
Either use a callback for a call that always succeeds or us a promise :)